packages/frontend/src/build/caching/JsonHttpClient.ts

```diff
@@ -33,7 +33,7 @@ export class JsonHttpClient {
   }
 }
 
-const TEN_MINUTES_IN_MS = 10 * 60 * 1000
+const THIRTY_MINUTES_IN_MS = 30 * 60 * 1000
```

Comment:
tbh, all the data that we currently have refreshes on hourly granularity, maybe make it to invalidate cache when hour is different ?

Reply:
yup probably better, will change